java/code_to_gast/java_expression.py

```diff
@@ -5,16 +5,36 @@
 gAST node of type funcCall
 '''
 def method_invocation_to_gast(node):
+    
     gast = {"type": "funcCall"}
     gast["args"] = java_router.node_to_gast(node.arguments)
     
     #TODO: change logic and add support for functions called on objects
     if node.qualifier == "System.out" and node.member == "println":
         gast["value"] = {"type": "logStatement"}
     else:
-        gast["value"] = {"type": "customStatement"}
+        # function not called on object
+        if not node.qualifier:
```

Comment:
Nit: remove not and flip the if else. I find that it improves readability when if statements aren't negated